test/integration/httpServer.test.js

```diff
@@ -19,4 +19,29 @@ describe('health', function () {
     expect(actualResult.status).to.equal(200)
     expect(actualResult.body).to.deep.equal(expectedResult)
   })
+
+  test('POST Order - 201', async function () {
```

Comment:
I think `order` tests should be in their own file like `attachment.test.js` and `recipe.test.js`